package.json

```
@@ -32,7 +32,9 @@
       "ts",
       "js"
     ],
-    "testRegex": "/src/__tests__/test\\..+\\.ts$",
+    "testRegex": [
+      "/src/__tests__/(.*/)?test\\..+\\.ts$"
+    ],
```

what about this so we are ready for more subfolders?

```suggestion
    "testRegex": "/src/__tests__/(.*/)*test\\..+\\.ts$",
```

although it's strange, does this work?

```json
    "testRegex": "/src/__tests__/**/test\\..+\\.ts$",
```

---

no, the latter does not work

---

but this does work:

`"testRegex": "/src/__tests__/(.*/)*test\\..+\\.ts$",`

---

No, it does not work, I already tested.

```error
SyntaxError: Invalid regular expression: //src/__tests__/**/test\..+\.ts$/: Nothing to repeat
    at new RegExp (<anonymous>)
    at /Users/jmillan/src/rtp.js/node_modules/@jest/core/build/SearchSource.js:149:46
    at Array.map (<anonymous>)
    at regexToMatcher (/Users/jmillan/src/rtp.js/node_modules/@jest/core/build/SearchSource.js:149:29)
    at new SearchSource (/Users/jmillan/src/rtp.js/node_modules/@jest/core/build/SearchSource.js:215:18)
    at /Users/jmillan/src/rtp.js/node_modules/@jest/core/build/runJest.js:265:16
    at Array.map (<anonymous>)
    at runJest (/Users/jmillan/src/rtp.js/node_modules/@jest/core/build/runJest.js:264:34)
    at startRun (/Users/jmillan/src/rtp.js/node_modules/@jest/core/build/cli/index.js:479:35)
    at runWithoutWatch (/Users/jmillan/src/rtp.js/node_modules/@jest/core/build/cli/index.js:494:10)
```